.circleci/config.yml

```diff
@@ -10,6 +10,13 @@ test-ci-lib-llvm: &test-ci-lib-llvm
 
 version: 2
 jobs:
+  x86-64-unknown-linux-gnu-nightly:
+    docker:
+      - image: ponylang/pony-ci:x86-64-unknown-linux-gnu-nightly
+    steps:
+      - checkout
+      - run: bash .ci-scripts/x86-64-unknown-linux-gnu-nightly.bash 1cc732891a064c6542e3632a6ee281b69dbdce18
+
```

Comment:
This "on every CI" is temporary and will be removed once I verify is it working.

The API key on line 18 will be replaced with an encrypted value.